app/parse.py

```diff
@@ -15,8 +31,91 @@ class Product:
     num_of_reviews: int
 
 
+def click_more_button(driver) -> None:
+    try:
+        more_button = driver.find_elements(By.CLASS_NAME, "ecomerce-items-scroll-more")
+        if more_button:
+            more_button = more_button[0]
+            style = more_button.get_attribute("style")
+            while not style:
+                more_button.click()
+                time.sleep(1)
```

Comment:
https://selenium-python.readthedocs.io/waits.html#explicit-waits
try to use WebDriverWait

Reply:
i understood that actually i don`t need time.sleep)